users.json

```diff
@@ -0,0 +1,1423 @@
+{
```

Comment:
sorry but I can't make up my mind for why we have a 1,423 lines of spec for a simple preferences API, SCIM may be an inspiration to us but we can't just take all of it